lib/wireprotocol/2_6_support.js

```diff
@@ -275,94 +198,54 @@ var setupClassicFind = function(bson, ns, cmd, cursorState, topology, options) {
     ignoreUndefined: ignoreUndefined
   });
 
-  // Set query flags
-  query.slaveOk = readPreference.slaveOk();
-
-  // Set up the option bits for wire protocol
-  if (typeof cmd.tailable === 'boolean') {
-    query.tailable = cmd.tailable;
-  }
-
-  if (typeof cmd.oplogReplay === 'boolean') {
-    query.oplogReplay = cmd.oplogReplay;
-  }
-
-  if (typeof cmd.noCursorTimeout === 'boolean') {
-    query.noCursorTimeout = cmd.noCursorTimeout;
-  }
-
-  if (typeof cmd.awaitData === 'boolean') {
-    query.awaitData = cmd.awaitData;
-  }
-
-  if (typeof cmd.partial === 'boolean') {
-    query.partial = cmd.partial;
-  }
+  if (typeof cmd.tailable === 'boolean') query.tailable = cmd.tailable;
```

Comment:
I'd like to keep the braces if possible.

Reply:
sure

Reply:
out of curiosity, where do you draw the line? I notice you're not asking to introduce braces above for e.g. all of the one-liners in `setupClassicFind` like:
```
if (cmd.sort) findCmd['$orderby'] = cmd.sort;
```

Reply:
Well, taking a look at your [OP_MSG implementation](https://github.com/mongodb-js/mongodb-core/pull/276/files) I see that you use this form, as well as with braces, as well as looping over an array of properties for assigning. 

I think we can all be a little inconsistent, so maybe better if we all agree on a rule of thumb?